test/common.js

```diff
@@ -138,12 +138,16 @@ common.getTestConfig = function(config) {
   return mergeTestConfig(config);
 };
 
-common.getSSLConfig = function() {
+common.getSSLConfig = function(options) {
```

Comment:
Rather than passing in the `options` here, the common functions in here expect the caller to use `common.extend` to augment after the fact.

Reply:
How do you mean?

Reply:
`common.extend(common.getSSLConfig, { minVersion: ... })`